src/utils/bwellPersonFinder.js

```diff
@@ -57,7 +57,7 @@ class BwellPersonFinder {
         let linkedPersons = await databaseQueryManager.findAsync({ query: { 'link.target.reference': currentSubject }});
 
         // iterate over linked Persons (breadth search)
-        while (!foundPersonId && linkedPersons && await linkedPersons.hasNext()) {
+        while (!foundPersonId && await linkedPersons.hasNext()) {
```

Comment:
We should add a if (linkedPersons) check around this while loop

Reply:
linkedPersons won't be null. It's cursor even if the results are empty.